app/models/manageiq/providers/hawkular/datawarehouse_manager.rb

```diff
@@ -38,38 +39,42 @@ def self.verify_ssl_mode
     end
 
     # Hawkular Client
-    def self.raw_connect(hostname, port, token, type = :alerts)
-      klass = case type
+    def self.raw_connect(options)
+      options[:type] ||= :alerts
+      options[:tenant] ||= '_system'
```

Comment:
```ruby
type = options.fetch(:type) { :alerts }
tenant = options.fetch(:tenant) { '_system' }
```

This is the "more correct" way of doing this, because if the user sets the key to `nil` or `false` they mean it, and we such treat it as such.